src/components/common/DeleteButton.tsx

```diff
@@ -49,7 +49,7 @@ const DeleteButton: FC<Props> = ({ itemIds, color, id, type, onClick }) => {
         aria-label={text}
         id={id}
         color={color}
-        type={type}
+        type={type as ActionButtonVariant}
```

Comment:
Is this caused by the nullable? If yes, consider using a non-nullable prop type for `type`, or returning `null` if not set (but I think removing the nullable makes sense).